activation/atxdb.go

```diff
@@ -146,9 +141,8 @@ func (db *ActivationDb) CalcActiveSetFromView(a *types.ActivationTx) (uint32, er
 
 }
 
-// ValidateAtx ensures the following conditions apply, otherwise it returns an error.
+// SyntacticallyValidateAtx ensures the following conditions apply, otherwise it returns an error.
 //
-// - No other ATX exists with the same MinerID and sequence number.
 // - If the sequence number is non-zero: PrevATX points to a valid ATX whose sequence number is one less than the
```

Comment:
"syntactically valid"; here and under positioning ATX
Also below, one of the conditions is that the NIPST verifies (to check this condition, we need to know the challenge, which is what written below)